pkg/cmd/serve/serve.go

```diff
@@ -161,40 +148,25 @@ func serveRun(ctx context.Context, cmd *cobra.Command, args []string, datastoreO
 		[]float64{.006, .010, .018, .024, .032, .042, .056, .075, .100, .178, .316, .562, 1.000},
 	))
 
-	middleware := grpc.ChainUnaryInterceptor(
-		requestid.UnaryServerInterceptor(requestid.GenerateIfMissing(true)),
-		logmw.UnaryServerInterceptor(logmw.ExtractMetadataField("x-request-id", "requestID")),
-		grpclog.UnaryServerInterceptor(grpczerolog.InterceptorLogger(log.Logger)),
-		otelgrpc.UnaryServerInterceptor(),
-		grpcauth.UnaryServerInterceptor(auth.RequirePresharedKey(token)),
-		grpcprom.UnaryServerInterceptor,
-		servicespecific.UnaryServerInterceptor,
-	)
+	if len(config.UnaryMiddleware) == 0 || len(config.StreamingMiddleware) == 0 {
```

Comment:
What if you really don't want any middleware?

Reply:
You can wipe them out if you want to:

```go
server, _ := config.Complete()
server.SetMiddleware(nil, nil)
```